Tensile/Source/lib/include/Tensile/ContractionProblem.hpp

```diff
@@ -99,10 +99,11 @@ namespace Tensile
          */
         struct BoundIndex
         {
-            BoundIndex(size_t xa=0, size_t xb=0) : a(xa), b(xb) {};
```

Comment:
The mirrored dimensions need to be incorporated into the `operationIdentifier` in some way in order for kernels to be differentiated properly.

Reply:
Does e7a2dd2d748893eaf63090009b33f0496f606614 cover all places where `operationIdentifier` is generated/used?

Reply:
That covers places where it's generated.  I'd recommend deleting the `addAMirrorDim` and `addBMirrorDim` functions since the operationIdentifier is generated at construction time and the ContractionProblem object is otherwise immutable.

Reply:
Done in 1e5a78aa12b162850dd90e37b6361c89ec0fb4c2